.github/workflows/pull-request.yml

```
@@ -240,36 +288,67 @@
-    if: github.actor != 'dependabot[bot]'
-    needs: [Test-Docusaurus-Deployment]
     steps:
-      - name: Checkout Repository
+      - name: Checkout
         uses: actions/checkout@v4
-      - name: Validate CodeRabbit.ai Approval
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)